src/Jackett.Common/Definitions/pixelados.yml

```diff
@@ -97,13 +100,36 @@ search:
   fields:
     category:
       selector: category_id
-    title:
+    types:
+      selector: types
+    megapack:
+      selector: types:contains(MEGAPACK)
+      optional: true
```

Comment:
megapack is not referenced as `.Result.megapack` anywhere. Redundant or missing usage?

Reply:
I was using to have a PACK tag, but is redundant because the packed TV Show always have Season.

Reply:
Added commit with changes